### PR TITLE
[Transform] Add apply option (select tensors)

### PR DESCRIPTION
**Pull request by gichan-jang**
User can select tensors to apply the transformation.
    
Signed-off-by: Gichan Jang <gichan2.jang@samsung.com>

Self evaluation:
1. Build test: [ *]Passed [ ]Failed [ ]Skipped
2. Run test: [ *]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: gst/nnstreamer/tensor_transform/tensor_transform.c [hunk 1]
```diff
@@ -798,6 +804,24 @@ gst_tensor_transform_set_property (GObject * object, guint prop_id,
...
+      gchar **strv = g_strsplit_set (param, ",", -1);
+      gint i, num = g_strv_length (strv);
+
+      for (i = 0; i < num; i++) {
+        val = g_ascii_strtoll (strv[i], NULL, 10);
+        if (errno == ERANGE) {
+          ml_loge ("Overflow occured during converting %s to a gint64 value",
+              strv[i]);
+        }
+        filter->apply = g_list_append (filter->apply, GINT_TO_POINTER (val));
```
Comment (jaeyun-jung) [inline gst/nnstreamer/tensor_transform/tensor_transform.c:820]: filter->apply is leak, plz add free function in finalize function.

Comment (gichan-jang) [reply gst/nnstreamer/tensor_transform/tensor_transform.c:820]: Updated.

File: gst/nnstreamer/tensor_transform/tensor_transform.c [hunk 2]
```diff
@@ -60,7 +60,7 @@
  * @brief Macro for debug mode.
  */
 #ifndef DBG
-#define DBG (!filter->silent)
+#define DBG TRUE
```
Comment (jaeyun-jung) [inline gst/nnstreamer/tensor_transform/tensor_transform.c:63]: plz revert this option

Comment (gichan-jang) [reply gst/nnstreamer/tensor_transform/tensor_transform.c:63]: Oooops..! Thanks

File: gst/nnstreamer/tensor_transform/tensor_transform.c [hunk 3]
```diff
@@ -798,6 +804,24 @@ gst_tensor_transform_set_property (GObject * object, guint prop_id,
...
       break;
+    case PROP_APPLY:
+    {
+      gint64 val;
+      const gchar *param = g_value_get_string (value);
+      gchar **strv = g_strsplit_set (param, ",", -1);
+      guint i, num = g_strv_length (strv);
+
+      for (i = 0; i < num; i++) {
+        val = g_ascii_strtoll (strv[i], NULL, 10);
```
Comment (dongju-chae) [inline gst/nnstreamer/tensor_transform/tensor_transform.c:815]: You should set `errno == 0` before calling those functions setting `errno` internally.

Comment (dongju-chae) [reply gst/nnstreamer/tensor_transform/tensor_transform.c:815]: Also, `ERANGE` only handles overflows. You should check other violations. Please check the glib document.

Comment (gichan-jang) [reply gst/nnstreamer/tensor_transform/tensor_transform.c:817]: Updated.

Comment (dongju-chae) [reply gst/nnstreamer/tensor_transform/tensor_transform.c:817]: Well.. you didn't provide `endptr` as an argument. So, the third check is meaningless.

Comment (gichan-jang) [reply gst/nnstreamer/tensor_transform/tensor_transform.c]: Right.. I missed it. Updated.

**Review by dongju-chae — APPROVED** [review]
LGTM

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**